### PR TITLE
Move glslify dependency

### PR DESCRIPTION
**Pull request by ibgreen**

### CODE REVIEW

**Comment by ibgreen** [comment]
@gnavvy @mike-marcacci FYI

File: package.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
     "geojson-normalize": "0.0.1",
     "get-pixels": "^3.3.0",
     "gl-matrix": "^2.3.2",
+    "glslify": "^5.0.0",
```
Comment (gnavvy) [inline package.json]: explicitly use ^5.1.0 here?

File: CHANGELOG.md [hunk 2]
```diff
@@ -25,6 +25,9 @@ All notable changes to deck.gl will be documented in this file.
 - Remove separate attribute definitions, simplifying layer subclass creation
 - Standar
 
+#### [2.4.8] - TBD
+- Move glslify to "core" dependencies in package.json
```
Comment (gnavvy) [inline CHANGELOG.md]: is "core" dependencies the same as dependencies?

**Comment by ibgreen** [comment]
Fixes #48

**Comment by mike-marcacci** [comment]
Thanks @ibgreen!

**Pull request merged**